packages/block-library/src/blocks/supernova/style.scss

```diff
@@ -37,3 +37,9 @@
 .supernova {
   @include layout-grid;
 }
+
+
+// Temporary !!!!!
```

Comment:
is it temporary or does it need to stay?

Reply:
It was temporary for me to better check stacked cards.